### DISCUSSION TITLE
Office URI Schemes on Mac

### DISCUSSION THREAD

**Discussion by TobiasWolters** [MicrosoftDocs/office-developer-client-docs#669] [Q&A]
We are using Office URI Schemes to open documents from our web-application.

In particular, we are doing the following:
```javascript
window.location.href = 'ms-word:ofe|u|<path-to-public-word-document>';
```

It works perfectly for both Word, Excel, and Powerpoint on Windows in all browsers.
However, it has stopped working on Mac in at least Chrome, Edge, and Safari with the newest version of MacOS and Office 365 ProPlus.

(see https://stackoverflow.com/questions/64880406/problem-opening-office-uri-scheme-urls-in-office-js-on-mac-safari for a more detailed description)

Let me know if there is a better place to post this bug report.

**Comment by ralphtheninja** [comment]
I'll post my workaround here as well. This works in our add-in on all platforms.

```js
window.open('ms-word:ofe|u|https://foo.sharepoint.com/path/to/file.docx')
```
